adapters/tlsio_openssl.c

```diff
@@ -934,7 +935,14 @@ static int load_cert_crl_http(
         goto error;
     }
 
+    LOCK_RESULT lockResult = Lock(crl_cache_lock);
```

Comment:
>crl_cache_lock [](http://example.com/codeflow?start=34&length=14)

Just to check is tlsio_openssl_init where the lock is initialzed called before this always ?

Reply:
@amitkumarshukla